virtual-desktop/src/app/authentication-manager/authentication-manager.service.ts

```diff
@@ -240,6 +240,14 @@ export class AuthenticationManager {
     });
   }
 
+  performPasswordReset(username: string, password: string, newPassword: string): Observable<Response> {
+    return this.http.post(ZoweZLUX.uriBroker.serverRootUri('auth-password'),
```

Comment:
you need a way to specify which plugin here.

Reply:
serviceHandler was added by @sakeerthy